packages/domain/tests/todos/todo.tests.js

```diff
@@ -177,6 +163,199 @@ describe("Todos.Todo", function() {
           })
         ]);
     });
+  });
+
+
+  describe("removing many todos", function() {
```

Comment:
Wondering if we can refine this test to better explain it's intent. It was helpful for debugging the issue with the dependency, but reading it without any context doesn't make much sense. What are we really testing here?